acra-censor/acra-censor_test.go

```diff
@@ -712,12 +783,16 @@ func TestConfigurationProvider(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	if len(handlers_) != 3{
```

Comment:
`acraCensor.handlers` and then we don't need to return all handlers just for tests